### ISSUE TITLE
TypeError: Can't convert 'NoneType' object to str implicitly

### ISSUE THREAD

**Issue by eigan** [wwwsevolod/SublimeLiveRsync#1]
This error pops up in console when running command: live_rsync_upload_full

```
Writing file /path/to/my/project.sublime-project with encoding UTF-8 (atomic)
Traceback (most recent call last):
  File "/Users/Einar/Library/Application Support/Sublime Text 3/Packages/SublimeLiveRsync/LiveRsync.py", line 60, in callback
    RsyncLogSingletone().log('Manual upload project:\n' + window.project_file_name())
TypeError: Can't convert 'NoneType' object to str implicitly
```

My Project file

``` JSON
{
    "folders":
    [
        {
            "follow_symlinks": true,
            "path": ".",
            "live_rsync": {
                "destination": "einar@devserver:~/public_html/devel/project",
                "ssh": true,
                "full_update_on_start" : true
            }
        }
    ]
}
```

**Comment by wwwsevolod** [comment]
hello, thank you, u r the first known user of this plugin :smiley_cat:
ill take a look at error in next few days

**Comment by eigan** [comment]
Love it! :)

Think i found the error.

print(window.project_file_name()) always returned None. So the project settings file was not found. Didnt know I had to do Project -> Save Project as. After I did that, window.project_file_name() started returning the path to the projectfile and SublimeLiveRsync started working :-)

It will anyway always fail (and output to console) if there is no project settings file. So I think you might want to add some checking on the project variable:

LiveRsync.py, line 30

``` python
if project != None and project != self.project:
```
